### ISSUE TITLE
New leaderboard page (aka "partner page")

### ISSUE THREAD

**Issue by dgquail** [hotosm/tasking-manager#6369]
Ticket to track [partner pages](https://www.missingmaps.org/partners/accenture) setup within Tasking Manager

**Comment by dgquail** [comment]
Hello Emilio!
We have a question about how to obtain the current projects of each partner (carousel component). We suppose that we should obtain a list with the ids of those projects to invoke the summary and tasks endpoints, but we cannot find how to obtain that list of current projects



![Image](https://github.com/hotosm/tasking-manager/assets/87376859/c7a3e374-073d-4375-bf92-0bf851fe2aa1)

Thanks!

**Comment by ramyaragupathy** [comment]
@emi420 - for now we can simply let the partnerships team populate project IDs for each partner without running actual validation for existence of projects. Like agreed we can have the second phase of the project to build the association between partners.

**Comment by emi420** [comment]
Ok @ramyaragupathy @dgquail , we'll need to add:

- A new column to the [database schema](https://github.com/hotosm/tasking-manager/issues/6346) named `current_projects` or something similar for a list of project IDs.
- A new field in the partner add/edit form for the ids
- Update the partner's endpoint

And then we'll use those ids in the front-end for the carousel.

**Comment by CristianAmici** [comment]
![Image](https://github.com/hotosm/tasking-manager/assets/72893031/35229dd4-c771-4b94-8612-e95ce72083c9)

We've introduced the "Current Project" section, we offer a slider display highlighting each active project. Furthermore, there's a zoom-enabled map focusing on the project's layer, granting users the ability to navigate and zoom in and out effortlessly. We've retained the same buttons and functionalities from the previous page for consistency.

**Comment by CristianAmici** [comment]
![Image](https://github.com/hotosm/tasking-manager/assets/72893031/100c36cb-314e-428b-85b6-8cc13a61bea5)

In this section, we present activity using three charts. We have employed the same bar components used in the projects section to create the list of participants, maintaining the link and previous functionality.

**Comment by CristianAmici** [comment]
![Image](https://github.com/hotosm/tasking-manager/assets/72893031/705a6a44-7bc1-4f86-9979-cc2f1f59dee9)

Hi! This is the resources section, and we would need to know if the resource buttons are partner-specific and how to dynamically retrieve that information for each partner.

**Comment by dgquail** [comment]
@emi420 @ramyaragupathy Cristian left a question for you

**Comment by ramyaragupathy** [comment]
> We've introduced the "Current Project" section, we offer a slider display highlighting each active project. Furthermore, there's a zoom-enabled map focusing on the project's layer
looks great!

> we would need to know if the resource buttons are partner-specific 
yes theses are partner specific and there could be different set of resources for each partner. @nicolelaine could you please confirm the usage of resources section?

cc @emi420

**Comment by emi420** [comment]
@CristianAmici The partners table has a field for "Other links" , max 5, it's called `website_links` in the DB schema and in the DTO:

https://github.com/hotosm/tasking-manager/issues/6346#issue-2250972477
https://github.com/hotosm/tasking-manager/issues/6346#issuecomment-2077392441

CC @ramyaragupathy @nicolelaine @dgquail

**Comment by dgquail** [comment]
@emi420 
ahh ok There was a mailto, that's why we thought it was other data. No problem, then we display the buttons based on that partner attribute

**Comment by nicolelaine** [comment]
@ramyaragupathy Yes, these resources are partner specific. I think 5 sounds like a good number.

**Comment by nicolelaine** [comment]
> ![Image](https://private-user-images.githubusercontent.com/72893031/327553344-35229dd4-c771-4b94-8612-e95ce72083c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ5ODk1MTksIm5iZiI6MTcxNDk4OTIxOSwicGF0aCI6Ii83Mjg5MzAzMS8zMjc1NTMzNDQtMzUyMjlkZDQtYzc3MS00Yjk0LTg2MTItZTk1Y2U3MjA4M2M5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA2VDA5NTMzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczMWY4ZjlkMzczMGVhOTRhYTQ1MGRmNzRjY2Y2ZmYwYWYxNzQ2MjhmYjk0MGY0MDNiNTc1MzE2YmRlNzMzMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ELxgfb0pP59NLSvzrLKdh4L--GLHqDVOKhyh3-tk3Ns)
> 
> We've introduced the "Current Project" section, we offer a slider display highlighting each active project. Furthermore, there's a zoom-enabled map focusing on the project's layer, granting users the ability to navigate and zoom in and out effortlessly. We've retained the same buttons and functionalities from the previous page for consistency.


@CristianAmici Is it possible to also see the progress bar on each project on the page? 

So, for example, how you can see that this project is currently 88% complete?

![Screenshot 2024-05-06 at 11 58 23](https://github.com/hotosm/tasking-manager/assets/15011784/1d262d18-8d72-494e-b728-5f61b1ab9f9e)

**Comment by CristianAmici** [comment]
![Image](https://github.com/hotosm/tasking-manager/assets/72893031/598f18b6-7ceb-4334-b9dd-696e527f7b9c)

@nicolelaine Hi! I forgot the percentage line that was supposed to go inside each slide. Yes, we implemented it and left the information as the project component takes it from a hover (ProjectProgressBar), as the information of the 3 fields comes from the API: "percentMapped", "percentValidated", "percentBadImagery", we implemented that bar that has the 3 colors for them. It seemed to us the most appropriate, but tell us if it's okay this way, and thanks for reminding me.
